markets/perps-market/test/integration/helpers/settleHelper.ts

```diff
@@ -1,75 +1,50 @@
 import { ethers } from 'ethers';
 import { Systems } from '../bootstrap';
-import { fastForwardTo } from '@synthetixio/core-utils/utils/hardhat/rpc';
-
-type IncomingChainState = {
-  systems: () => Systems;
-  provider: () => ethers.providers.JsonRpcProvider;
-};
 
 export type SettleOrderData = {
-  keeper: () => ethers.Signer;
-  marketId: () => ethers.BigNumber;
-  accountId: () => number;
-  feedId: () => string;
-  startTime: () => number;
-  settlementDelay: () => number;
-  offChainPrice: () => ethers.BigNumberish;
-};
-
-type SettleOrderReturn = {
-  settleTx: () => ethers.ContractTransaction;
+  systems: () => Systems;
+  keeper: ethers.Signer;
+  marketId: ethers.BigNumber;
+  accountId: number;
+  offChainPrice: ethers.BigNumberish;
+  settlementTime: number;
+  feedId: string;
 };
 
-type SettleOrderType = (data: SettleOrderData, chainState: IncomingChainState) => SettleOrderReturn;
-
-export const settleOrder: SettleOrderType = (data, chainState) => {
-  let tx: ethers.ContractTransaction;
-  let updateFee: ethers.BigNumber;
-
-  let pythPriceData: string, extraData: string;
-
-  before('fast forward to settlement time', async () => {
```

Comment:
mostly removed the before blocks and made all of it top level and so it runs inside a before block